### ISSUE TITLE
wt stat confusion for "number of key/value pairs"

### ISSUE THREAD

**Issue by ddanderson** [wiredtiger/wiredtiger#1733]
After creating a btree with 279 entries, wt dump shows 279 entries, and wt stat reports 281 for `btree: number of key/value pairs`.  The difference arises because entries in internal pages that point to subordinate pages are counted in this statistic.  The btree for this case looks like this:
```
       |
  Internal page
   /        \
Leaf       Leaf
```
We show:
```
  btree: number of key/value pairs=281
  btree: row-store internal pages=1
  btree: row-store leaf pages=2
```
To get the "right" answer in a row store with current stat output, one needs to calculate: `E - I - L + 1`, where:
```
E  == btree: number of key/value pairs
I  == btree: row-store internal pages
L  == btree: row-store leaf pages
```
Or are there more factors to consider to get an accurate view?  At any rate, this seems confusing and makes it harder to use wt stat as a quick diagnostic tool. It came up as part of SUPPORT-1248.

**Comment by sueloverso** [comment]
Thanks for this issue Don because I read Dan's edited entry in that ticket last night and was in the middle of composing email to ask what was going on there because it bugged me because I thought it was accurate and may have misled them earlier.  Thanks for figuring that out!  I think that this situation is going to be common. Looking at tables in a copied database directory, quick-turnaround debugging situation.

However, your calculation is off somewhere, `E + I + L - 1` gives me 281 + 1 + 2 - 1 = 283.

**Comment by ddanderson** [comment]
Oops.  I think E - I - L + 1.  Each pointer to a leaf page is double counted, Each pointer to a internal page is double counted.  One (the head node) is not double counted.

**Comment by ddanderson** [comment]
Closed by merge of #1752.
